examples/project.yaml

```diff
@@ -8,42 +8,16 @@ spec:
   eventSubscriptions:
   - source: brigade.sh/bitbucket
     types:
-    - repo:forked
-    - pullrequest:created
-    - pullrequest:updated
-    labels:
```

Comment:
We must have created this example before qualifiers were a thing. The events from this gateway are _qualified_ by repo.